src/org/pahappa/systems/services/IncidentServiceImpl.java

```diff
@@ -44,7 +44,36 @@ public Incident saveIncident(Incident incident) throws Exception {
 
 	@Override
 	public Incident updateIncident(Incident incident) throws Exception {
-		// TODO Auto-generated method stub
+		//Check if both the comment and title sections have been filled
+		if (incident.getTitle().isEmpty()){
+			throw new ValidationFailedException("Please enter title");
+		}else if (incident.getTitle()==null){
+			throw new ValidationFailedException("Please enter title");
+		}else if (incident.getComment().isEmpty()){
```

Comment:
@ttcollins this can lead to NullException Errors. You cannot check for empty strings before checking for nullability.